java/adapter/avro/src/main/java/org/apache/arrow/consumers/AvroNullConsumer.java

```diff
@@ -20,18 +20,18 @@
 import java.io.IOException;
 
 import org.apache.arrow.vector.FieldVector;
-import org.apache.arrow.vector.ZeroVector;
+import org.apache.arrow.vector.NullVector;
 import org.apache.avro.io.Decoder;
 
 /**
  * Consumer which consume null type values from avro decoder.
  * Corresponding to {@link org.apache.arrow.vector.ZeroVector}.
  */
-public class AvroNullConsumer implements Consumer<ZeroVector> {
+public class AvroNullConsumer implements Consumer<NullVector> {
 
-  private ZeroVector vector;
+  private NullVector vector;
 
-  public AvroNullConsumer(ZeroVector vector) {
```

Comment:
should there be an increment on NullVector count when consuming?

Reply:
Right, fixed.